dotCMS/src/main/java/com/dotcms/rest/api/v1/apps/ExportSecretForm.java

```diff
@@ -46,10 +47,11 @@ public boolean isExportAll() {
 
     @Override
     public String toString() {
-        final List<String> stringsList = appKeysBySite.entrySet().stream()
-                .map(entry -> "Site " + entry.getKey() + " keys: " + String
-                        .join(",", entry.getValue())).collect(
-                        Collectors.toList());
+        final List<String> stringsList =
+                appKeysBySite == null ? ImmutableList.of() : appKeysBySite.entrySet().stream()
```

Comment:
This logic should have a test

Reply:
it's a toString I used to print info. But I got rid of the piece that used this code. 
I think I'll get rid of it too.

Reply:
done